oregon/templates/home.html

```diff
@@ -38,54 +38,9 @@
rs" alt="Cottage Grove newspapers" /><area shape="rect" coords="122,227,198,244" href="http://oregonnews.uoregon.edu/search/pages/results/?lccn=sn97071002&amp;lccn=sn97071001&amp;lccn=sn97071003&amp;dateFilterType=yearRange&amp;date1=1846&amp;date2=1922&amp;ortext=&amp;andtext=&amp;phrasetext=&amp;proxtext=&amp;proxdistance=5&amp;rows=20&am" title="Springfield newspapers" alt="Springfield newspapers" /><area shape="rect" coords="69,227,120,244" href="http://oregonnews.uoregon.edu/search/pages/results/?lccn=sn96088145&amp;lccn=sn96088146&amp;lccn=sn83025130&amp;date1=1848&amp;date2=1922&amp;date1=&amp;date2=&amp;sequence=&amp;ortext=&amp;andtext=&amp;phrasetext=&amp;proxtext=&amp;proxdistance=5&amp;rows=20&amp;searchType=advanced" title="Eugene newspapers" alt="Eugene newspapers" /><area shape="rect" coords="45,163,90,181" href="http://oregonnews.uoregon.edu/search/pages/results/?lccn=sn85033162&amp;date1=1848&amp;date2=1922&amp;date1=&amp;date2=&amp;sequence=&amp;ortext=&amp;andtext=&amp;phrasetext=&amp;proxtext=&amp;proxdistance=5&amp;rows=20&amp;searchType=advanced" title="Toledo newspapers" alt="Toledo newspapers" /><area shape="rect" coords="113,138,161,153" href="http://oregonnews.uoregon.edu/search/pages/results/?lccn=sn99063954&amp;lccn=sn99063957&amp;lccn=sn99063955&amp;lccn=sn99063956&amp;lccn=sn99063952&amp;date1=1848&amp;date2=1922&amp;date1=&amp;date2=&amp;sequence=&amp;ortext=&amp;andtext=&amp;phrasetext=&amp;proxtext=&amp;proxdistance=5&amp;rows=20&amp;searchType=adva" title="Salem newspapers" alt="Salem newspapers" /><area shape="rect" coords="144,104,221,125" href="http://oregonnews.uoregon.edu/search/pages/results/?lccn=sn84022661&amp;lccn=sn84022662&amp;dateFilterType=yearRange&amp;date1=1846&amp;date2=1922&amp;ortext=&amp;andtext=&amp;phrasetext=&amp;proxtext=&amp;proxdistance=5&amp;rows=20&amp;searchType=advance" title="Oregon City newspapers" alt="Oregon City newspapers" /><area shape="rect" coords="141,74,198,89" href="http://oregonnews.uoregon.edu/search/pages/results/?lccn=sn83025138&amp;lccn=sn84022673&amp;lccn=sn85042444&amp;lccn=sn83045780&amp;lccn=sn00063676&amp;lccn=sn83045782&amp;date1=1848&amp;date2=1922&amp;date1=&amp;date2=&amp;sequence=&amp;ortext=&amp;andte" title="Portland newspapers" alt="Portland newspapers" /><area shape="rect" coords="47,83,112,101" href="http://oregonnews.uoregon.edu/search/pages/results/?lccn=sn99063850&amp;dateFilterType=yearRange&amp;date1=1846&amp;date2=1922&amp;ortext=&amp;andtext=&amp;phrasetext=&amp;proxtext=&amp;proxdistance=5&amp;rows=20&amp;searchType=advanced&amp;sort=date" title="Tillamook newspapers" alt="Tillamook newspapers" /><area shape="rect" coords="100,39,166,55" href="http://oregonnews.uoregon.edu/search/pages/results/?lccn=sn97071109&lccn=sn93051654&lccn=2004260421&lccn=2004260419&dateFilterType=yearRange&date1=1846&date2=2013&ortext=&andtext=&phrasetext=&proxtext=&proxdistance=5&rows=20&searchType=advanced" title="St. Helens newspapers" alt="St. Helens newspapers" />
```

Comment:
Actually I think I see what changed here.  The formatting definitely needs a fix, as it's really only a minor change to a single `<area>` tag.  But I'd also like to see the "map changes" commit pulled out of this PR and put into a new PR or just master directly.  Unless I'm totally crazy, that commit is an unrelated fix to the homepage map.